packages/tbds-forms/lib/form-text-input.scss

```diff
@@ -0,0 +1,16 @@
+.tbds-form-text-input {
+  appearance: none;
+  background-color: #fff;
+  border: 1px solid #bbb;
+  border-radius: 5px;
+  margin-bottom: 1em;
+  padding: 0.5em;
+  transition-duration: 200ms;
+  transition-property: border-color;
+  width: 100%;
```

Comment:
Is it useful to use any of the support vars here?

Reply:
Yes, I think so! But I held off for the moment, as I think it'll be easier to go through and extract a bunch at once, so you can see how they need to be systemized. I started to create a few, but felt like I was backing myself into a corner quickly. I figured a bit of repetition upfront is fine.